src/pages/home/features/Features.spec.tsx

```diff
@@ -0,0 +1,100 @@
+import harness from '@dojo/framework/testing/harness';
+import { tsx } from '@dojo/framework/widget-core/tsx';
+import { add } from '@dojo/framework/has/has';
+import * as css from './Features.m.css';
+import Features from './Features';
+import Card from '../../../widgets/card/Card';
+
+describe('Features', () => {
+	it('renders', () => {
+		add('build-time-render', true, false);
```

Comment:
Out of curiosity what does this do here? Do we need the build-time-render flag for this unit test since the `Features` widget never checks it?

Reply:
Yep, not needed either, shall remove